go/vt/sqlparser/ast.go

```diff
@@ -52,6 +52,7 @@ type (
 		SetLimit(*Limit)
 		SetLock(lock Lock)
 		MakeDistinct()
+		ContainsOnlyInformationSchema() bool
```

Comment:
We should name it as SystemSchema as it can be other than information_schema as well.

Reply:
Done